### PR TITLE
DlTextArea - add states

### PR DESCRIPTION
**Pull request by denfedweb**
**Thank you for your contribution to the repo. Before submitting this PR, please make sure:**
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests
- [ ] You have updated documentation
- [x] You have tested your changes

**Please provide a description of the changes proposed in the pull request and reference any related issues in the repository.**

**Please indicate if this PR contains:**
- [ ] A bug fix
- [ ] A feature request
- [ ] Breaking changes
- [x] An enhancement

### CODE REVIEW

File: src/components/essential/DlTextArea/DlTextArea.vue [hunk 1]
```diff
@@ -113,9 +167,37 @@ export default defineComponent({
...
+        },
+        topMessage: {
+            type: String,
+            default: ''
+        },
+        hideClearButton: {
+            type: Boolean,
+            default: false
+        },
+        clearButtonTooltip: {
```
Comment (fadiDL) [inline src/components/essential/DlTextArea/DlTextArea.vue:195]: this is a string. check how it behaves in dl-input today

File: src/components/essential/DlTextArea/DlTextArea.vue
Comment (fadiDL) [inline src/components/essential/DlTextArea/DlTextArea.vue:1]: Missing Props: 
- redAsterisk
- readonly
- dense
- counterReverse

**Review by fadiDL — CHANGES_REQUESTED** [review]

File: src/components/essential/DlTextArea/DlTextArea.vue (hunk 1 above)
Comment (denfedweb) [reply src/components/essential/DlTextArea/DlTextArea.vue:195]: image

Comment (fadiDL) [reply src/components/essential/DlTextArea/DlTextArea.vue:203]: Opening a new issue about this. thanks!

**Review by fadiDL — APPROVED** [review]

**Pull request merged**